### PR TITLE
Removed Common parameters

### PR DESCRIPTION
**Pull request by v-akarke**
Removed Common parameters and tested all the examples; everything is working as expected.

### CODE REVIEW

File: module/EntraBeta/customizations/Get-EntraBetaPolicy.ps1 [hunk 1]
```diff
@@ -102,9 +58,8 @@
          
         $data = $response | ConvertTo-Json -Depth 50 | ConvertFrom-Json        
         $respList = @()
-       
-        foreach ($res in $data) {                                  
-            switch ($res.type) {
+        foreach ($res in $data) {
+        switch ($res.type) {
```
Comment (KenitoInc) [inline module/EntraBeta/customizations/Get-EntraBetaPolicy.ps1:62]:
```suggestion
            switch ($res.type) {
```

File: module/EntraBeta/customizations/Set-EntraBetaPolicy.ps1 [hunk 2]
```diff
@@ -76,56 +76,13 @@
...
+            Write-Debug("============================ TRANSFORMATIONS ============================")
+            $params.Keys | ForEach-Object {"$_ : $($params[$_])" } | Write-Debug
+            Write-Debug("=========================================================================`n")
             
-                $body = $params | ConvertTo-Json
-                Invoke-GraphRequest -Headers $customHeaders -Uri $uri -Body $body -Method $Method
-
+            $body = $params | ConvertTo-Json
+            Invoke-GraphRequest -Headers $customHeaders -Uri $uri -Body $body -Method $Method
+
```
Comment (KenitoInc) [inline module/EntraBeta/customizations/Set-EntraBetaPolicy.ps1:85]: Remove this line

File: module/EntraBeta/customizations/Get-EntraBetaPolicy.ps1 (hunk 1 above)
Comment (v-akarke) [reply module/EntraBeta/customizations/Get-EntraBetaPolicy.ps1]: Updated

File: module/EntraBeta/customizations/Set-EntraBetaPolicy.ps1 (hunk 2 above)
Comment (v-akarke) [reply module/EntraBeta/customizations/Set-EntraBetaPolicy.ps1]: Removed line

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**